modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -93,9 +96,8 @@ public class OAuth2ServiceAbstractIntegrationTest extends ISIntegrationTest {
 	/**
 	 * Initialize
 	 *
-	 * @param userMode
-	 *            - User Id
-	 * @throws Exception
+	 * @param userMode - User Id
+	 * @throws Exception Exception
```

Comment:
Shall we fix the formatting by adding missing full stops? Le's check other places also

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096